src/Cameras/babylon.arcRotateCamera.ts

```diff
@@ -634,7 +634,7 @@
                 this._targetHost = null;
                 this._target = newTarget;
                 this._targetBoundingCenter = null;
-                this.onMeshTargetChangedObservable.notifyObservers(null);
+                this.onMeshTargetChangedObservable.notifyObservers(<any>null);

```

Comment:
This is crappy.
I will restore the Nullable on observable even if I already know it will trigger dozen of warnings

Reply:
I agree that this is not the best solution. WHat happens if you define the type of this Observer to `AbstractMesh | null` instead of returning the Nullable to the observers? Haven'T checked, but it might work.